lib/voicebase/v2/client.rb

```diff
@@ -79,6 +133,22 @@ def get_media_progress(args = {}, headers = {})
         ), api_version)
       end
 
+      def delete_file(args = {}, headers = {})
```

Comment:
Thanks for the suggestion. Juergen used this pattern through the code and I'm not feeling like changing all those places now. We plan to clean up this gem to be better suited for public availability, so perhaps that's a better time to make improvements like this.